### PR TITLE
Update event-triggers.md

### PR DESCRIPTION
**Pull request by shendrekbharath**
On hasura/graphql-engine repo, under event-triggers.md found couple of broken links mentioned below:

https://github.com/hasura/graphql-engine/blob/master/community/examples/serverless-push
https://github.com/hasura/graphql-engine/blob/master/community/examples/serverless-etl

Replaced the broken links with the right ones:
https://github.com/hasura/graphql-engine/tree/master/community/sample-apps/serverless-push
https://github.com/hasura/graphql-engine/tree/master/community/sample-apps/serverless-etl

### CODE REVIEW

File: event-triggers.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Trigger push notifications and emails based on database events. Try the demo and
 
 * [Watch demo](https://www.youtube.com/watch?v=nuSHkzE2-zo)
 * [Try it out](https://serverless-push.demo.hasura.app/)
-* [Tutorial](community/examples/serverless-push)
+* [Tutorial](https://github.com/hasura/graphql-engine/tree/master/community/sample-apps/serverless-push)
```
Comment (shahidhk) [inline event-triggers.md]:
```suggestion
* [Tutorial](community/sample-apps/serverless-push)
```

File: event-triggers.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ Transform and load data into external data-stores. Check out this demo and tutor
 
 * [Watch demo](https://youtu.be/kWVEBWdEVAA)
 * [Try it out](https://serverless-etl.demo.hasura.app/)
-* [Tutorial](community/examples/serverless-etl)
+* [Tutorial](https://github.com/hasura/graphql-engine/tree/master/community/sample-apps/serverless-etl)
```
Comment (shahidhk) [inline event-triggers.md]:
```suggestion
* [Tutorial](community/sample-apps/serverless-etl)
```

**Comment by marionschleifer** [comment]
@shendrekbharath can you commit @shahidhk's suggestions please, so that we can merge? 🙂

**Comment by shendrekbharath** [comment]
> @shendrekbharath can you commit @shahidhk's suggestions please, so that we can merge? 🙂

I have applied @shahidhk's suggestions, please validate.

**Pull request merged**